WooCommerce.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -73,15 +55,6 @@
           "version": "1.1.1"
         }
       },
-      {
-        "package": "Nimble",
-        "repositoryURL": "https://github.com/Quick/Nimble.git",
-        "state": {
-          "branch": null,
-          "revision": "7795df4fff1a9cd231fe4867ae54f4dc5f5734f9",
-          "version": "13.7.1"
-        }
-      },
```

Comment:
I'm not sure why these dependencies disappeared from the workspace resolved file in this commit. I would have expect that to happen as part of #15612 🤔